__tests__/app.test.js

```diff
@@ -0,0 +1,54 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed.js");
+
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed({
+    articleData,
+    commentData,
+    topicData,
+    userData,
+  });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  test("GET 200 - respond with an object containing all the slug", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const { result } = res.body;
+        //if result is an array
+        expect(result).toBeInstanceOf(Array);
+        //length to be 3
+        expect(result.length).toBe(3);
+        //for each slug object should match the test object
+        result.forEach((slugTopic) => {
+          expect(slugTopic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+  test("GET 404 - if not found", () => {
```

Comment:
API endpoint is not exclusive to api/topics, therefore should be in its own describe block. Also the example endpoint could therefore be considered brittle. Could another endpoint that doesnt exists not belong to api/topics be used?